lib/fork.js

```diff
@@ -153,7 +153,14 @@ module.exports = function (file, opts, execArgv) {
 		return promise;
 	};
 
+	var isExitInitiated = false;
```

Comment:
Unfortunately, I forgot... I remember it had something to do with this https://github.com/avajs/ava/blob/ea25e802bce67b195d5f0fc331291bb02f16b866/api.js#L305. Sure, I'll use `onetime`.